Automattic/wp-calypso
# Reader Discover - Add 'dailyprompt' to the list of interest tags / tabs

Related to # p58i-fGN-p2#comment-59877

## Proposed Changes

* Adds the dailyprompt tag to the list of tags to render tabs for in the Discover section of the reader.
* We simply add the data for the dailyprompt tag if it is not present in the list. Alternatively we could update the read/interests endpoint to always return dailyprompt at the front of the list for every locale, but this endpoint is used in other areas and that behavior may not be suitable everywhere so for now filtering this in on the front-end seems to make sense.

## Testing Instructions



* Load discover
* Verify the Daily Prompt tag exists after first-posts and latest
* Verify the stream loads as expected and there are no regressions.

After:

<img width="916" alt="Screenshot 2023-10-27 at 12 19 28 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/3289308a-112d-4b18-8b0b-46762ee5c4f3">


## Pre-merge Checklist
